Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/fragments/CameraFragment.kt

```diff
@@ -62,8 +60,8 @@ import kotlin.coroutines.suspendCoroutine
 
 class CameraFragment : Fragment() {
 
-    /** AndroidX navigation arguments */
-    private val args: CameraFragmentArgs by navArgs()
+    /** Lowest Camera setting for recording video */
+    private lateinit var lowestSetting: CameraInfo
```

Comment:
Maybe rename to "cameraSetting" and have the comment say "Camera setting to use when recording video".
While it is also the lowest available setting, for the purposes of most of the other code, the important thing is that this stores setting we are using to record video, not that it is the lowest available setting.

Reply:
okay, makes sense.